src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,123 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75F;
```

Comment:
```suggestion
    private static final int INITIAL_CAPACITY = 16;
    private static final float LOAD_FACTOR = 0.75F;
```
We can use 'DEFAULT' in these names if we have the possibility to change these values (in specific constructor for example). If we can not change it - it is not DEFAULT_INITIAL_CAPACITY, it is simple INITIAL_CAPACITY